Pset-1.py

```diff
@@ -3,5 +3,5 @@ def temp_converter(fahrenheit):
     return 5*(fahrenheit-32)/9 
```

Comment:
Add Comments

Reply:
Add Docstring

Reply:
I have made the  necessary changes accordingly